### PR TITLE
Clarify action 8/9 signal TM external triggers.

### PR DESCRIPTION
**Pull request by timsifive**
Also be more consistent in using "TM external triggers" in definition of tmexttrigger.

Addresses #783

### CODE REVIEW

File: Sdtrig.tex [hunk 1]
```diff
@@ -86,13 +86,19 @@ \section{Actions}
 \hline
 5 & Reserved for use by the trace specification. \\
 \hline
-8 -- 9 & Signal the firing of the trigger to other blocks within the hart (e.g. as countable events to hpmcounters).  Use external debug trigger output 0 or 1 (respectively). \\
+8 -- 9 & Send a signal to TM external trigger output 0 or 1 (respectively).  the
+firing of the trigger to blocks. \\
```
Comment (pdonahue-ventana) [inline Sdtrig.tex:90]: This second sentence looks like an editing error.

File: Sdtrig.tex [hunk 2]
```diff
@@ -86,13 +86,19 @@ \section{Actions}
...
 \hline
 other & Reserved for future use. \\
 \hline
 \end{tabular}
 \end{table}
 
+\begin{commentary}
+    Actions 8 and 9 are intended to increment custom event counters, but these
+    signals could also be brought out to pins for use by external logic.
```
Comment (pdonahue-ventana) [inline Sdtrig.tex:99]: Maybe this is just me but I think that `s/out to pins/to outputs/` would be better.

File: Sdtrig.tex (hunk 1 above)
Comment (timsifive) [reply Sdtrig.tex:90]: Thanks.

File: Sdtrig.tex (hunk 2 above)
Comment (timsifive) [reply Sdtrig.tex:99]: That's more general. Thanks.

**Pull request merged**